### DISCUSSION TITLE
streamload 50个并发提交的问题

### DISCUSSION THREAD

**Discussion by linlianghong** [apache/doris#19496] [RFC]
**问题**
通过streamload进行50个线程并发，每个线程会有500的数据量去提交，在测试环境中有ABC三台服务器（同网段），A部署了FE以及BE，BC分别部署BE，在B中部署了发送程序到A:8030，同时50个线程并发处理是可以处理得过来。
但是发布到生产，生产的配置为ABC三台服务器（同网段），D（不同网段），A部署了FE以及BE，BC分别部署BE，在D部署了发送程序发送到A:8030，刚启动那会的速度是正常，几秒钟过后出现请求time_wait的情况，查询A服务的日志，查看到插入表出现了write lock（写入被锁）的异常，但是测试环境并没有出现该异常，而且测试环境提交的数据长度比生产的还要大，压测的量更多，但是都没出现这种情况，请问这是啥原因
![image](https://github.com/apache/doris/assets/41034640/92898890-c388-4ee9-ae84-a3c05cd765a8)

**Comment by linlianghong** [comment]
**补充**
生产还有出现failed to call frontend servic(请求前端服务失败的异常)
![image](https://github.com/apache/doris/assets/41034640/5fcd025e-9530-4473-bc7a-44c563f7a351)

**Reply by LemonLiTree** [reply]
which version?

**Reply by linlianghong** [reply]
1.2.3

**Reply by LemonLiTree** [reply]
Can be tested in 1.2.4
